tests/MongoDB.Driver.Legacy.Tests/Builders/IndexKeysBuilderTests.cs

```diff
@@ -219,5 +254,17 @@ public void TestTextIndexCreation()
             Assert.Equal("spanish", index["default_language"].AsString);
             Assert.Equal(1, index["key"]["c"].AsInt32);
         }
+
+        [Fact]
+        public void Wildcard_should_return_expected_result()
+        {
+            var keys = IndexKeys.Wildcard();
+            BsonDocument expected = BsonDocument.Parse("{ \"$**\" : 1 }");
+            keys.ToBsonDocument().Should().Be(expected);
+
+            keys = IndexKeys.Wildcard("b");
+            expected = BsonDocument.Parse("{ \"b.$**\" : 1 }");
+            keys.ToBsonDocument().Should().Be(expected);
+        }
     }
 }
```

Comment:
Minor nit: recommend a newline at the end of the file :)

Reply:
Done